pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -1128,14 +1134,6 @@ func buildDefaultPassthroughCluster(env *model.Environment, proxy *model.Proxy)
 		ConnectTimeout:       gogo.DurationToProtoDuration(env.Mesh.ConnectTimeout),
 		LbPolicy:             lbPolicyClusterProvided(proxy),
 	}
-	passthroughSettings := &networking.ConnectionPoolSettings{
-		Tcp: &networking.ConnectionPoolSettings_TCPSettings{
-			// The envoy default is 1024. This isn't configurable right now so we set
-			// this to a very high value so outbound connections are not limited.
-			MaxConnections: 1024 * 100,
```

Comment:
Why remove? Don't we have a 1024 by default?

Reply:
This was overriding the defaults above. I accidentally removed the call to `applyConnectionPool()` too, though - which meant defaults were not applied. I fixed that. The default passthrough cluster now has the same CircuitBreaker settings as other clusters